src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,84 @@
+.stop-watch__block {
+  margin: 200px auto;
+}
+
+.stop-watch {
+  position: relative;
+  width: 400px;
+  height: 400px;
+
+  border: 4px solid;
+  border-radius: 50%;
+
+  &__number {
+
+    margin: 0;
+
+    &--12 {
+      position: absolute;
+      left: 50%;
+
+      transform: translateX(-50%);
+    }
+
+    &--3 {
+      position: absolute;
+      left: 100%;
+      top: 50%;
+
+      transform: translate(-100%, -50%);
+    }
+
+    &--6 {
+      position: absolute;
+      left: 50%;
+      top: 100%;
+
+      transform: translate(-50%, -100%);
+    }
+
+    &--9 {
+      position: absolute;
+      top: 50%;
```

Comment:
Im not sure, but maybe better to add postition to .stop-watch__number class< just for "no-repeat" it in all modificators, anyway it the same block.